app/editor/src/store/hooks/content/useContent.ts

```diff
@@ -42,13 +42,34 @@ export const useContent = (props?: IContentProps): [IContentState, IContentContr
       return result;
     },
     addContent: async (content: IContentModel) => {
-      return await dispatch('add-content', () => api.addContent(content));
+      const result = await dispatch('add-content', () => api.addContent(content));
+      if (state.content) {
```

Comment:
I don't think `state.content` would be positive if a user was adding their first content item.

Reply:
I believe it still returns something like:
`
...
items: [],
total: 0
`
when there are no items

Reply:
Hmm ok works again, but found a weird issue with `publishedOn` that happens occasionally